server/src/main/java/org/candlepin/model/Pool.java

```diff
@@ -101,6 +101,9 @@
         /** Attribute used to determine which specific host the pool was created for */
         public static final String REQUIRES_HOST = "requires_host";
 
+        /** Attribute used to determine if a pool was created by sharing */
+        public static final String SHARE = "share";
```

Comment:
I'm actually going to drop the SHARE attribute and am going to replace that with a pool type, `SHARE_DERIVED`.